src/TaskRunner.php

```diff
@@ -58,29 +74,73 @@ public function run()
 			$start  = Time::now();
 			$output = null;
 
+			$this->cliWrite("Processing : " . ($task->name ?: "Task"), 'green');
+
 			try
 			{
 				$output = $task->run();
+
+				$this->cliWrite("Executed   : " . ($task->name ?: "Task"), "cyan");
 			}
 			catch (\Throwable $e)
 			{
+				$this->cliWrite("Failed     : " . ($task->name ?: "Task"), "red");
+
 				log_message('error', $e->getMessage(), $e->getTrace());
+
 				$error = $e;
 			}
 			finally
 			{
 				// Save performance info
-				$this->performanceLogs[] = new TaskLog([
+				$taskLog = new TaskLog([
 					'task'     => $task,
 					'output'   => $output,
 					'runStart' => $start,
 					'runEnd'   => Time::now(),
 					'error'    => $error,
 				]);
+
+				// Save performance log to the database
+				$taskLog->saveToDatabase();
+
+				$this->performanceLogs[] = $taskLog;
```

Comment:
I had the same question while writing my PR. I don't know If you need the logs in the performanceLogs Array or not.
In my implementation I'm using TaskLog to manage all Tasks & Performance data to avoid adding any methods in the Task class.

Is it ok to delete it ?